src/pages/Map.js

```diff
@@ -1,40 +1,44 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import GoogleMapReact from 'google-map-react';
 import { API_KEY } from '../config.js';
-import { Icon } from '@iconify/react';
-import location12Filled from '@iconify-icons/fluent/location-12-filled';
 import './Map.css';
 import LocationPin from '../components/LocationPin.js';
+import { LocationsContext } from '../context/LocationsContext';
 
-const location = {
-  lat: 45,
-  lng: -123.456,
-};
+export default function Map() {
+  const locationsContext = useContext(LocationsContext);
 
-const zoomLevel = 12;
+  //set locations, zoomLevel & coordinates based on context
+  const locations = locationsContext.locations;
+  const zoomLevel = locationsContext.zoomLevel;
+  const coordinates = locationsContext.coordinates;
+
+  const _onChildClick = (key) => {
```

Comment:
Curious why you chose to start this function name with an `_`? That’s usually something programmers do to indicate a function is private. In the case of components, everything’s pretty well encapsulated, so it’s not strictly necessary.

Reply:
I think the example we were looking at was in typescript!